### PR TITLE
Update .NET Framework targeting pack version

### LINKED ISSUE #1137: Reference Assemblies for .NET Framwork 3.5
Are there any plans to add reference assemblies for .NET Framework 3.5 into `Microsoft.NETFramework.ReferenceAssemblies`?

### PR DESCRIPTION
**Pull request by dsplaisted**
Update to newly released version of [Microsoft.NETFramework.ReferenceAssemblies](https://www.nuget.org/packages/Microsoft.NETFramework.ReferenceAssemblies/)

Fixes https://github.com/microsoft/dotnet/issues/1137

@NikolaMilosavljevic @AArnott

### CODE REVIEW

**Review by joeloff — APPROVED** [review]

File: src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.props [hunk 1]
```diff
@@ -101,7 +101,7 @@ Copyright (c) .NET Foundation. All rights reserved.
     <DebugSymbols Condition="'$(DebugSymbols)'==''">false</DebugSymbols>
     <CheckForOverflowUnderflow Condition="'$(CheckForOverflowUnderflow)'==''">false</CheckForOverflowUnderflow>
     <AutomaticallyUseReferenceAssemblyPackages Condition="'$(AutomaticallyUseReferenceAssemblyPackages)'==''">true</AutomaticallyUseReferenceAssemblyPackages>
-    <MicrosoftNETFrameworkReferenceAssembliesLatestPackageVersion>1.0.0</MicrosoftNETFrameworkReferenceAssembliesLatestPackageVersion>
+    <MicrosoftNETFrameworkReferenceAssembliesLatestPackageVersion>1.0.1</MicrosoftNETFrameworkReferenceAssembliesLatestPackageVersion>
```
Comment (jnm2) [inline src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.props:104]: This version has a bug and @NikolaMilosavljevic is working on publishing a fix.

Comment (AArnott) [reply src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.props:104]: Can you share a link to the bug? I'm very curious.

Comment (rstm-sf) [reply src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.props:104]: https://github.com/microsoft/dotnet/issues/1137#issuecomment-833107242

Comment (jnm2) [reply src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.props:104]: https://www.nuget.org/packages/Microsoft.NETFramework.ReferenceAssemblies/1.0.2 is now published.

**Pull request merged**